napari_svg/layer_to_xml.py

```diff
@@ -44,6 +44,79 @@ def map(self, image):
 }
 
 
+def layer_transforms_to_xml_string(meta):
+    """Get the xml representation[1]_[2]_ of the layer transforms.
+
+    Parameters
+    ----------
+    meta : dict
+        The metadata tuple from the layer.
```

Comment:
Nit: It's a dictionary, not a tuple, and that's already in the type specifier above, so remove?

```suggestion
        The metadata from the layer.
```